Master-OB-OpenAPI.json

```
@@ -11647,6 +11650,23 @@
             }
           }
         ]
+      },
+      "ProdID": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementString"
+          },
+          {
+            "type": "object",
+            "description": "Unique identifier for the Product. ",
```

```suggestion
            "description": "Unique identifier for the Product.",
```
Just some trailing whitespace to remove.